core/src/test/scala/unit/kafka/log/OffsetIndexTest.scala

```diff
@@ -165,6 +168,14 @@ class OffsetIndexTest extends JUnitSuite {
     assertEquals("Full truncation should leave no entries", 0, idx.entries)
     idx.append(0, 0)
   }
+
+  @Test
+  def forceUnmapTest(): Unit = {
+    val idx = new OffsetIndex(nonExistantTempFile(), baseOffset = 0L, maxIndexSize = 10 * 8)
```

Comment:
It is an existing method, but perhaps name could be changed to `nonExistentTempFile`?